composer.json

```diff
@@ -22,12 +22,14 @@
         "source": "https://github.com/mayflower/mo4-coding-standard"
     },
     "require-dev": {
+        "phpstan/phpstan": "^0.9",
+        "phpstan/phpstan-strict-rules": "^0.9",
         "phpunit/phpunit": "^4.0 || ^5.0 || ^6.0 || ^7.0"
     },
     "require": {
-        "squizlabs/php_codesniffer": "^3.2.3",
-        "escapestudios/symfony2-coding-standard": "^3.0",
+        "php": ">=7.1.0",
```

Comment:
version constraints without upper restrictions are considered harmful as it allows future breaking releases of PHP by default: https://getcomposer.org/doc/faqs/why-are-unbound-version-constraints-a-bad-idea.md

Reply:
fixed.